EIPS/eip-4762.md

```diff
@@ -251,6 +230,13 @@ When a **write** event of `(address, sub_key, leaf_key)` occurs, perform the fol
 
 Note that tree keys can no longer be emptied: only the values `0...2**256-1` can be written to a tree key, and 0 is distinct from `None`. Once a tree key is changed from `None` to not-`None`, it can never go back to `None`.
 
+Note that values should only be added to the witness if there is sufficient gas to cover their associated event costs.
+
+`CREATE*` and `*CALL` reserve 1/64th of the gas before the nested execution. Note that, following the behavior of access lists, the 1/64th of the gas is subtracted:
+
+ * **BEFORE** charging the witness costs when performing a `CALL`, `CODECALL`, `DELEGATECALL` or`STATICCALL`
```

Comment:
Want to use `*CALL` to also contemplate `AUTHCALL`? (Also note a missing space after `or`)